apps/phone/src/os/apps/hooks/useApps.tsx

```diff
@@ -8,6 +8,7 @@ import { useSettingsValue } from '../../../apps/settings/hooks/useSettings';
 import { IconSetObject } from '@typings/settings';
 import { useRecoilValue } from 'recoil';
 import { phoneState } from '@os/phone/hooks/state';
+import { extname } from 'path';
```

Comment:
Unused import

Reply:
Again same... never added this but ill remove

Reply:
removed